runtime/doc/lua.txt

```diff
@@ -968,16 +968,35 @@ paste({lines}, {phase})                                          *vim.paste()*
                     |paste|
 
 schedule_wrap({cb})                                      *vim.schedule_wrap()*
-                Defers callback `cb` until the Nvim API is safe to call.
+                Wraps callback `cb` in a function which will defer the execution
+                of `cb` until the Nvim API is safe to call.
+
+                The user must call the returned function themselves if it is not
+                being used as an argument to a function that will call it for them,
+                such as `vim.fn_defer`.
+
+                Example: Using as an argument >
+
+                 local safe = vim.schedule_wrap(function()
+		   vim.api.nvim_command('echomsg "test"')
+                 end)
+                 vim.defer_fn(safe, 100)
+
+                Example: Using on it's own >
```

Comment:
typo: "its"

Reply:
I followed the capitalization example from other examples,

https://github.com/taylan97/neovim/blob/959349b688c282bbead31067d62507d63d1732e6/runtime/doc/lua.txt#L498

https://github.com/taylan97/neovim/blob/959349b688c282bbead31067d62507d63d1732e6/runtime/doc/lua.txt#L942

Though others do not capitalize after the colon,

https://github.com/taylan97/neovim/blob/959349b688c282bbead31067d62507d63d1732e6/runtime/doc/lua.txt#L480

For datas sake, more examples start with a capital than do not.

```
~/neovim/runtime/doc $ rg --no-heading "Example: [A-Z]" | wc -l
26 (-3, three of these are "TCP", "ISO" and "UTF")

~/neovim/runtime/doc $ rg --no-heading "Example: [a-z]" | wc -l
17
```

(do not fret about the lack of examples, there are 456 examples that simply read  "Example: >")

I do not say this to be pedantic to "win the argument", only to point out perhaps another potential (very minor) doc style fault (can open another pull request but maybe it would conflict too much with upstream vim I don't know how that works).

I will alter it to be lower cased, most online rules seem to say this is most accepted as "correct, or at least 'not incorrect'".

Reply:
That's fine, no need to worry about it. It is indeed a matter of style (which cannot be expected to be consistent in a document grown over decades). I wouldn't have mentioned it had I not already noticed the typo ;)